frontend/src/pages/LogsPage/useGetLogs.ts

```diff
@@ -36,41 +30,35 @@ export const useGetLogs = ({
 	startDate,
 	endDate,
 	disablePolling,
+	sortColumn = 'timestamp',
+	sortDirection = Types.SortDirection.Desc,
 }: {
 	query: string
 	project_id: string | undefined
 	logCursor: string | undefined
 	startDate: Date
 	endDate: Date
 	disablePolling?: boolean
+	sortColumn?: string
+	sortDirection?: Types.SortDirection
 }) => {
-	// The backend can only tell us page info about a single page.
-	// It has no idea what pages have already been loaded.
-	//
-	// For example: say we make the initial request for 100 logs and hasNextPage=true and hasPreviousPage=false
-	// That means that we should not make any requests when going backwards.
-	//
-	// If the user scrolls forward to get the next 100 logs, the server will say that hasPreviousPage is `true` since we're on page 2.
-	// Hence, we track the initial information (where "window" is effectively multiple pages) to ensure we aren't making requests unnecessarily.
-	const [windowInfo, setWindowInfo] = useState<PageInfo>(initialWindowInfo)
 	const [loadingAfter, setLoadingAfter] = useState(false)
-	const [loadingBefore, setLoadingBefore] = useState(false)
-
-	useEffect(() => {
-		setWindowInfo(initialWindowInfo)
-	}, [query, startDate, endDate])
 
 	const { data, loading, error, refetch, fetchMore } = useGetLogsQuery({
 		variables: {
 			project_id: project_id!,
 			at: logCursor,
-			direction: Types.SortDirection.Desc,
+			direction: sortDirection,
```

Comment:
This is tangental but needed to swap over the console log table which is ordered by timestamp ASC